app/views/foreman_ansible/config_reports/_ansible.html.erb

```diff
@@ -1,6 +1,11 @@
 <%= webpacked_plugins_js_for :foreman_ansible %>
 <%= stylesheet 'foreman_ansible/foreman-ansible' %>
+<% if ansible_run_in_check_mode?(logs.first) %>
```

Comment:
Having a banner is one of the requirements, so I would definitely keep it. Storing the information about check mode in log is just a way to have 'meta' information about the report so that we know if the banner should be displayed, so I think it is ok to hide it.